website/content/userguide/minio.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "setting up MinIO on gokrazy"
+weight: 50
+---
+
+# Setting up MinIO on gokrazy
+
+Since MinIO is a storage solution, one needs to have the permanent storage with
```

Comment:
Let’s make this a bit more actionable even without having to pull up the quickstart. How about:

Since MinIO is a storage solution, you need to enable permanent storage on your gokrazy installation by running the `mkfs` command that `gokr-packer` prints. For more details, see [Quickstart](/quickstart/).

Reply:
I like this a lot! Sounds less RTFM'y and way more practical 😄